### DISCUSSION TITLE
Any plans to add a keymap importer?

### DISCUSSION THREAD

**Discussion by entity** [zed-industries/zed#6763] [Feature Requests]
Hi folks,

I currently use a custom Jetbrains Keymap (not the default), is there any plans to implement an importer?

**Comment by entity** [comment]
Bump

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
